packages/jazz-react/src/index.tsx

```diff
@@ -18,7 +18,7 @@ import { AuthState, ReactAuthHook } from "./auth/auth.js";
 
 /** @category Context & Hooks */
 export function createJazzReactContext<Acc extends Account>({
-    auth: authHook,
+    auth: useAuthHook,
```

Comment:
Without the `use` prefix the compiler treats this function as a cacheable value, breaking the "order" rule of hooks